libs/api/domains/license-service/src/lib/utils/translations.ts

```diff
@@ -1,3 +1,16 @@
+import { Locale } from 'locale'
+import { LicenseLabelsObject } from '../licenceService.type'
+
+export const getLabel = (
+  labelKey: string,
+  locale: Locale,
+  label?: LicenseLabelsObject,
+) => {
+  return label
+    ? label[`${labelKey}`]
+    : Object.entries(i18n).find((x) => x[0] === `${labelKey}`)?.[1][locale]
```

Comment:
Love this change 😻 

Could we go with something a little more safe, something like this?

```suggestion
    : (i18n?.[labelKey]?.[locale] || '')
```

Reply:
Lol yes 🥇

Reply:
Or no, type error :( 

`Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ licenseNumber: { is: string; en: string; }; fullN`